src/i18n/locales/en-US.json

```diff
@@ -328,6 +326,7 @@
   "global.actions.dialogs.walletStateInvalid.title": "Invalid wallet state",
   "global.actions.dialogs.wrongPinError.message": "PIN is incorrect.",
   "global.actions.dialogs.wrongPinError.title": "Invalid PIN",
+  "global.attention": "Attention",
```

Comment:
FYI changing the key causes you to have to re-translate everything. For the extension I just keep the old key name if we ever have to rename something.

Reply:
mhm I know, but I thought "it's just 1 word, and this way code will be cleaner". I'll just keep the old key ;)